mod_reforged/hooks/skills/special/night_effect.nut

```diff
@@ -0,0 +1,42 @@
+::Reforged.HooksMod.hook("scripts/skills/special/night_effect", function(q) {
+	q.m.HitChancePerTile <- -10;
+
+	q.getTooltip = @(__original) function()
+	{
+		local ret = __original();
+		foreach (entry in ret)
+		{
+			if (entry.id == 12 && entry.icon == "ui/icons/ranged_skill.png")
+			{
+				entry.text = "Has " + ::MSU.Text.colorizePercentage(this.m.HitChancePerTile) + " chance to hit per tile of distance"
+			}
+		}
+		return ret;
+	}
+
+	q.onGetHitFactors = @(__original) function( _skill, _targetTile, _tooltip )
+	{
+		__original(_skill, _targetTile, _tooltip);
+
+		local malus = this.m.HitChancePerTile * (this.getContainer().getActor().getTile().getDistanceTo(_targetTile) - _skill.m.MinRange);
+		if (malus != 0)
+		{
+			_tooltip.push({
+				icon = "ui/tooltips/negative.png",
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizePercentage(malus) + " [Nighttime|Skill+night_effect]")
```

Comment:
Please add `{AddSign = false}` as the second argument to `colorizePercentage` because in hit factors we don't want to add sign (the sign is covered by the `icon`)